### PR TITLE
Build movie card component 

### LINKED ISSUE #23: create movie card component
create a movie card component that when hovering should show an orange layer with the name of the movie,
the component should receive the movie object as props and you will need to use (id, title, poster_path) and a handler function that should take the movie id
image

### PR DESCRIPTION
**Pull request by shadan72**
includes movie card and movie grid components with their styles
Fixes #23

### CODE REVIEW

File: src/components/ResultedMovies/MovieGrid.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  ];
+  return (
+    <div className="grid justify-center">
+      <div className="grid sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-12 ">
+        {movieData.map((movieItem, index) => (
+          <div
+            key={index}
+            className="flex items-center justify-center gradient-orange w-1/5 h-3/12 borderRadius-10  mb-4 rounded-sm"
+          >
+            <MovieCard singleMovie={movieItem} />
```
Comment (wisammechano) [inline src/components/ResultedMovies/MovieGrid.js:56]: The prop should be called movie

File: src/components/ResultedMovies/MovieGrid.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import MovieCard from '../../components/ResultedMovies/MovieCard';
+export default function MovieGrid() {
+  const movieData = [
```
Comment (wisammechano) [inline src/components/ResultedMovies/MovieGrid.js:4]: nice work using a placeholder

File: src/components/ResultedMovies/MovieCard.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export default function MovieCard(movie) {
```
Comment (wisammechano) [inline src/components/ResultedMovies/MovieCard.js:3]: the argument should either be props or a destructuring object like `{singleMovie}` or `{movie}` if you change the prop name

File: tailwind.config.js [hunk 4]
```diff
@@ -6,7 +6,46 @@ module.exports = {
         lightblue: '#18A0FB', // an example of extending tailwind
       },
     },
+    borderRadius: {
+      sm: '10px',
```
Comment (wisammechano) [inline tailwind.config.js:10]: no need to extend this as small is already 10 px

File: tailwind.config.js [hunk 5]
```diff
@@ -6,7 +6,46 @@ module.exports = {
...
+    },
+    width:{
+      '1/5':'257px'
```
Comment (wisammechano) [inline tailwind.config.js:13]: 1/5 is supposed to be percentage. wrong extending

File: tailwind.config.js [hunk 6]
```diff
@@ -6,7 +6,46 @@ module.exports = {
...
+    },
+    height:{
+      '3/12':'290px'
+    }
   },
   variants: {},
-  plugins: [],
+  plugins: [
+    function ({ addComponents }) {
```
Comment (wisammechano) [inline tailwind.config.js:21]: Why is this?

File: tailwind.config.js (hunk 5 above)
Comment (wisammechano) [reply tailwind.config.js:13]: revert back to normal, and use tailwind classes correctly

File: tailwind.config.js (hunk 6 above)
Comment (shadan72) [reply tailwind.config.js:21]: Because I needed customized style that I could not implemented with Tailwindcss classes.

**Pull request merged**